.vscode/launch.json

```diff
@@ -50,7 +50,15 @@
           "ignoreFailures": true
         }
       ]
-    }
+    },
+    //Example of a ROS Launch file
+    {
+      "name": "ROS: Launch File",
+      "type": "ros",
+      "request": "launch",
+      "preLaunchTask": "build",
+      "target": "${workspaceFolder}/install/share/${input:package}/launch/${input:ros_launch}",
```

Comment:
Does this need a merge install and isolated install entry?

Reply:
Good catch! Added the entry for isolated install